### PR TITLE
repo: Create Repository Templates

### PR DESCRIPTION
**Pull request by HappyNTH**
Adds two issue templates and a PR template, working towards #11 

Accompanies [API PR 854](https://github.com/BristolSTA/uobtheatre-api/pull/854)

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+**PR Steps**
+
+These steps are to ensure you correctly make a PR. _Please delete them before submitting the PR!_
+
+1. For almost all development, you should be targeting `develop` rather than `main`. Only releases target `main`!
```
Comment (zst-c) [inline .github/pull_request_template.md:5]: Are we going to put something here about "make sure you run `make pr` before merging to ensure the PR matches linting" or similar? Or is that more on the general help pages I guess?

**Review by zst-c — COMMENTED** [review]
Looks good - just left one slight query about the PR template!

**Comment by zst-c** [comment]
Worth a thought also as to whether we make organisation-wide templates, rather than specific repo ones, to make upkeep easier in the future? We can still choose to make some specific ones for web and api though, so as to specialise and override into the future. https://www.rhysmills.com/post/2021/09/07/set-a-default-pr-template-for-a-github-organisation.html

**Comment by HappyNTH** [comment]
> Worth a thought also as to whether we make organisation-wide templates, rather than specific repo ones, to make upkeep easier in the future? We can still choose to make some specific ones for web and api though, so as to specialise and override into the future. https://www.rhysmills.com/post/2021/09/07/set-a-default-pr-template-for-a-github-organisation.html

It seems definitely worth setting up a default repository, not least because we can place [other things](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file) in there later down the line (like an organisation-level readme) if we wish. However, what gives me pause is the following:

> If a repository has any files in its own `.github/ISSUE_TEMPLATE` folder, including issue templates or a _config.yml_ file, none of the contents of the default `.github/ISSUE_TEMPLATE `folder will be used.

Therefore, if we want to do any sort of customisation on this or the API repository, we'll need to replicate entirely. That being said, I think setting up some basics is the correct first step for now. I'm going to, therefore, create a _generic_ set of issue templates that will apply across everything, but still leave _specific_ PR templates for Web & API in these PRs. Coming to a re-review near you soon!

**Pull request merged**